server/libs/infra/src/entities/asset-face.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { AssetEntity } from './asset.entity';
+import { PersonEntity } from './person.entity';
+
+@Entity('asset_faces')
+export class AssetFaceEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string;
```

Comment:
This actually isn't needed. Asset id and person id should be the primary columns for this table.